frappe/public/js/frappe/list/base_list.js

```diff
@@ -204,6 +204,11 @@ frappe.views.BaseList = class BaseList {
 		};
 
 		if (frappe.boot.desk_settings.view_switcher) {
+			/* @preserve
+			for translation, don't remove
+			__("List View") __("Report View") __("Dashboard View") __("Gantt View"),
+			__("Kanban View") __("Calendar View") __("Image View") __("Inbox View"),
+			__("Tree View") __("Map View") */
 			this.views_menu = this.page.add_custom_button_group(__('{0} View', [this.view_name]),
```

Comment:
@surajshetty3416 any translation for the above strings won't get picked up by `__('{0} View')`, right?